src/main/java/yablunovskyi/bookstore/controller/AuthenticationController.java

```diff
@@ -15,8 +13,6 @@
 import yablunovskyi.bookstore.security.AuthenticationService;
 import yablunovskyi.bookstore.service.UserService;
 
-@RestController
-@Validated
```

Comment:
Why did you remove RestController annotation?

Reply:
By recommendation(. Returned it already